src/components/Well/Well.tsx

```diff
@@ -0,0 +1,32 @@
+import type { HeadingLevel } from '../../types/headingLevel';
+import List from '../List/List';
+import ListItem from '../List/ListItem';
+import './Well.less';
```

Comment:
Can the well Less file be deleted now that we know call-to-action links can just use the `m-list_item` class?

Reply:
Removed 👍🏾